ArduPlane/Parameters.cpp

```diff
@@ -290,6 +291,8 @@ const AP_Param::Info Plane::var_info[] = {
     // @Param: FENCE_TOTAL
     // @DisplayName: Fence Total
     // @Description: Number of geofence points currently loaded
+    // @Range: 0 100
```

Comment:
This seems wrong, you shouldn't be editing this directly, it should always be done via a GCS using the dedicated messages, rather then just reviving whatever was written out to storage before.

Reply:
does editing the metadata fields add a functional change here? if not, does it make sense to have this as an advanced param at all?

Reply:
Unless I've missed something with the expected usage `FENCE_TOTAL` is like `MISSION_TOTAL` or `RALLY_TOTAL`, it's in the param system because it was easy persistent storage, but it's not a parameter a user should edit directly, and should always be done via the dedicated protocol for transferring the items. (I don't think this should be shown on any of the lists actually)

Reply:
i was under the impression that adding the @Range and @Increment fields would allow for correct display only without actually making it editable. reverted to previous state. and thanks a lot for reviewing!